tetragonpod/PROJECT

```diff
@@ -0,0 +1,20 @@
+# Code generated by tool. DO NOT EDIT.
```

Comment:
kind of tempted to say also delete this file. do we need this file?

Reply:
This is a kubebuilder specific file, but since we are like integrating stuff at this point, the project might not be fully handled by kubebuilder so we can delete it.